src/containerTest/kotlin/com/quipalup/katydid/CreateMealLogEntryCT.kt

```diff
@@ -0,0 +1,73 @@
+package com.quipalup.katydid
+
+import io.restassured.RestAssured
+import io.restassured.module.kotlin.extensions.Extract
+import io.restassured.module.kotlin.extensions.Given
+import io.restassured.module.kotlin.extensions.Then
+import io.restassured.module.kotlin.extensions.When
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.web.server.LocalServerPort
+
+@SpringBootTest(
+    classes = [Katydid::class],
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT
+)
+class CreateMealLogEntryCT {
+
+    @LocalServerPort
+    private val port: Int = 0
+
+    @BeforeEach
+    fun setUp() {
+        RestAssured.port = port
+        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails()
+    }
+
+    @Test
+    fun `searches young humans`() {
+        Given {
+// TODO        contentType("application/vnd.api+json")
+            body(requestBody)
+        } When {
+            post("/log-entries")
+        } Then {
+// TODO            contentType("application/vnd.api+json")
+            statusCode(201)
+        } Extract {
+            `assert that response body is equal to`(body().asString(), expectedResponseBody)
+        }
+    }
+
+    private val requestBody: String = """
+            {
+              "data": 
+                  {
+                    "type": "log-entry",
```

Comment:
```suggestion
                    "type": "meal-log-entry",
```
Since this endpoint will accept more than one type (see [JSON:API spec about resource objects](https://jsonapi.org/format/#document-resource-objects)) of log entry (`meal-log-entry`, `pee-log-entry`, `poop-log-entry`...), we should rename that key.